web-ui/src/main/resources/catalog/locales/it-gnui.json

```diff
@@ -0,0 +1,27 @@
+{
+  "facets.block.title.OrgForResource": "Organizzazione",
+  "facets.block.title.availableInServices": "Available for",
+  "facets.block.title.cl_hierarchyLevel.key": "Resource type",
+  "facets.block.title.cl_maintenanceAndUpdateFrequency.key": "Frequenza di aggiornamento",
+  "facets.block.title.cl_spatialRepresentationType.key": "Spatial representation",
+  "facets.block.title.cl_status.key": "Stato",
+  "facets.block.title.creationYearForResource": "Anno di creazione",
+  "facets.block.title.resolutionScaleDenominator": "Scala",
+  "facets.block.title.tag": "Parole chiave",
+  "facets.block.title.tag.default": "Tag",
+  "facets.block.title.th_regions_tree.default": "Regioni",
+  "nav.back": "Back",
+  "record.action.download": "Scarica",
+  "record.action.view": "Vedere",
+  "record.more.details": "Read more",
```

Comment:
Ok so the english default value is already managed on the backend ?

Reply:
Transifex do that; it copy the "main" language to others. But we still may have the case where GN-UI translation has some keys that are not yet in GN4 API - so in that case it still make sense to have a fallback on GN-UI side to english.